### PR TITLE
Power State after Power Fail

### PR DESCRIPTION
**Pull request by filipleple**

### CODE REVIEW

File: docs/unified-test-documentation/dasharo-compatibility/power-after-fail.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    Menu.
+1. Enter the `Dasharo System Features` menu using the arrow keys and Enter.
+1. Enter the `Power Management Options` submenu.
+1. Verify the `Power state after power failure` field.
+
+**Expected result**
+
+The `Power state after power failure` field should inform that the current
+state of
+Secure Boot is `Powered Off`.
```
Comment (mkopec) [inline docs/unified-test-documentation/dasharo-compatibility/power-after-fail.md:35]: Copypaste error? :)

**Review by mkopec — CHANGES_REQUESTED** [review]
One small issue

File: docs/unified-test-documentation/dasharo-compatibility/power-after-fail.md (hunk 1 above)
Comment (filipleple) [reply docs/unified-test-documentation/dasharo-compatibility/power-after-fail.md]: Yes indeed, resolved in  [recent commit](https://github.com/Dasharo/docs/commit/f401a74c4b776a6989141b7a340943a1ddfa0294)

**Comment by mkopec** [comment]
Orphan page check fails - you need to add the file to `mkdocs.yml` so that it's available.

**Comment by mkopec** [comment]
@filipleple Pre-commit is failing. Please install pre-commit hooks as documented in README

**Review by mkopec — APPROVED** [review]

**Pull request merged**